basket/news/middleware.py

```diff
@@ -51,39 +34,3 @@ def __call__(self, request):
         response = self.get_response(request)
         response["X-Backend-Server"] = self.backend_server
         return response
-
-
-def is_ip_address(hostname):
-    return bool(IP_RE.match(hostname))
-
-
-class EnforceHostnameMiddleware(object):
```

Comment:
Is this no longer in use?

Reply:
I chatted with Brett and it appears to not be used.